### ISSUE TITLE
Port to WebExtension

### ISSUE THREAD

**Issue by s8321414** [gomita/tabscope#65]
Any chance to port this extension API, or it will stop working when Firefox 57 is out.

**Comment by grahamperrin** [comment]
[1385453 - Make it possible for async tab switcher to "warm up" tabs that are likely to be displayed soon - bugzilla.mozilla.org](https://bugzilla.mozilla.org/show_bug.cgi?id=1385453)

-  RESOLVED FIXED in Firefox 57
- referred from [Photon Engineering Newsletter #15 | Dolske's blog - dolske.wordpress.com](https://dolske.wordpress.com/2017/09/11/photon-engineering-newsletter-15/) (2017-09-11).

**Comment by ErikKrause** [comment]
This extenstion is desperatly needed on FF 57. Although there are dozens of extensions dealing with tabs there is none which is so useful. Please port it to webextension! Let me know if I can be of any help (testing etc.).

**Comment by davidhedlund** [comment]
> Although there are dozens of extensions dealing with tabs there is none which is so useful. Please port it to webextension!

@gomita That's true, your add-on is unique. Are you going to make a WebExtension of it?

**Comment by davidhedlund** [comment]
[Tip Tab](https://addons.mozilla.org/en-US/firefox/addon/tip-tab/) comes close but it requires a separate tab to be opened to zoom the thumbnail view for the other tabs. I asked the Tip Tab developer if there was any interest in implementing the Tab Scope functionality.
